### ISSUE TITLE
big picture mode steam play filter not working correctly

### ISSUE THREAD

**Issue by LiamDawe** [ValveSoftware/steam-for-linux#6893]
#### Your system information

* Steam client version (build number or date): Feb 11
* Distribution (e.g. Ubuntu): Manjaro
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:

Using the newly added big picture mode filter, to show only steam play white-listed titles doesn't work as intended. It shows a lot more. This was tested with steam play only turned on for "supported" titles as per the option, the other setting was left off (and yes I restarted Steam to be sure).

https://www.gamingonlinux.com/uploads/articles/article_media/15384328091581588953gol1.jpg

**Comment by austinp-valve** [comment]
Right now it's showing certified and user opted in games when the filter is active. The idea was that you'd enable the "use steam play with all games" option and then this filter would show the official whitelist and anything you've specifically whitelisted. I could tweak the localization but we will likely be reworking a lot of this as we go along.

**Comment by LiamDawe** [comment]
That does make a little sense, still, doesn't seem to be working like that. My shot as the example includes titles like Fieldrunners and Fieldrunners 2 which neither has any Steam Play set on them in my client (and never has). Same with Doki Doki, a game again I've never touched.

**Comment by alkazar** [comment]
Is this still a problem? The filter seems to work for me as @austinp-valve indicated.

**Comment by LiamDawe** [comment]
Yup, if the idea is to show whitelist + what I personally force SP on then yes it works fine. Closing.
